src/commands/componentPaths.js

```diff
@@ -16,9 +16,10 @@ export const desc = 'print out component paths for the given project';
 
 export { default as builder } from '../helpers/validateCommand';
 
-export const handler = (config) => {
+// This config parameter here is the arguments passed in through yargs.
+export const handler = async (config) => {
```

Comment:
unless we do drop node 6, this means we're transpiling async/await; i'd prefer to avoid that.

Reply:
it would be nice to drop support eventually, since it did reach eol.

Reply:
or can we use a babel transform?